ocs_ci/ocs/resources/mcg.py

```
@@ -861,26 +861,20 @@ def _get_mirroring_percentage():
-            except TimeoutExpiredError:
-                logger.error(
-                    f"The mirroring process is stuck from last {timeout} seconds."
-                )
-                assert False
-            mirror_percentage = _get_mirroring_percentage()
-        logger.info("All objects mirrored successfully.")
+        try:
+            for mirror_percentage in TimeoutSampler(
+                timeout, 30, _get_mirroring_percentage
+            ):
+                logger.info(f"Mirroring is {mirror_percentage}% done.")
```

Ruff (0.15.13)

[warning] Logging statement uses f-string

(G004)